app/controllers/internal/path_redirects_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Internal::PathRedirectsController < Internal::ApplicationController
+  layout "internal"
+
+  before_action :set_path_redirect, only: %i[edit update destroy]
+
+  after_action only: %i[update destroy create] do
+    Audit::Logger.log(:internal, current_user, params.dup)
+  end
+
+  def new
+    @path_redirect = PathRedirect.new
+  end
+
+  def create
+    @path_redirect = PathRedirect.new(new_path_redirect_params)
+
+    if @path_redirect.save
+      flash[:success] = "Path Redirect created successfully!"
+      redirect_to internal_path_redirects_path
+    else
+      flash[:danger] = @path_redirect.errors.full_messages.to_sentence
+      render new_internal_path_redirect_path
+    end
+  end
+
+  def index
+    @q = PathRedirect.order(created_at: :desc).ransack(params[:q])
```

Comment:
Is `@q` an elasticsearch convention?

Reply:
It's a [Ransack](https://docs.dev.to/internal/internal-search/) convention

Reply:
I believe that it is specific to ransack, but I'm not sure. Good q @vaidehijoshi -  I'm curious too. 😄